packages/data-models/tests/options/option-item.test.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-env jest */
 import OptionItem from '@/options/option-item'
+import StorageAdapter from '@/storages/storage-adapter.js'
```

Comment:
It is not very neccesary, I used StorageAdapter because it was widely used inside `options.test.js`.
I would fix according to other tests inside `option-item.test.js` and remove StorageAdapter import.

Reply:
thanks